Consul.Test/Consul.Test.csproj

```diff
@@ -22,10 +21,10 @@
     <PackageReference Include="Xunit.SkippableFact" Version="1.3.12" />
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.0.2" />
-    <PackageReference Include="GitHubActionsTestLogger" Version="1.1.0" />
+    <PackageReference Include="GitHubActionsTestLogger" Version="1.2.0" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
```

Comment:
I think that `TargetFrameworkIdentifier` better communicates the intent and avoids repeating the framework's version.

Reply:
OK.

Can't find much docs about `TargetFrameworkIdentifier` apparently.